politeiad/cmd/politeia/README.md

```diff
@@ -102,13 +102,16 @@ Update record: 9dfe084fccb7f27c0000
 
 You can update the status of an unvetted record using one of the following
 statuses:
-- `publish` - make the record a public, vetted record.
-- `censor` - keep the record unvetted and mark as censored.
+- `censor`     - keep the record unvetted and mark as censored.
+- `public`     - make the record a public, vetted record.
+- `unreviewed` - make the record be in a unreviewed changes status.
```

Comment:
You don't need to include this since its not something the user can set.  A record is unreviewd on submittal, but you can't manually set the status to unreviewed.